src/app/components/tile/index.html

```diff
@@ -68,25 +68,33 @@
         <stache-code>componentType</stache-code> — The class type for the tile component.
       </li>
       <li>
-        <stache-code>providers</stache-code> — An array of data providers that can be passed to the tile.
+        <stache-code>providers</stache-code> — <em>(Optional.)</em> An array of data providers that can be passed to the tile.
       </li>
     </ul>
     </sky-demo-page-property>
       <sky-demo-page-property
         propertyName="layout"
       >
-      A <stache-code>SkyTileDashboardConfigLayout</stache-code> object that describes the layout of the tile dashboard. The <stache-code>SkyTileDashboardConfigLayout</stache-code> object contains a <stache-code>singleColumn</stache-code> property of type <stache-code>SkyTileDashboardConfigLayoutColumn</stache-code> and a <stache-code>multiColumn</stache-code> property that is an array of type <stache-code> SkyTileDashboardConfigLayoutColumn</stache-code>. The <stache-code>singleColumn</stache-code> layout displays on small screens, while the <stache-code>multiColumn</stache-code> layout displays on larger screens. The <stache-code>SkyTileDashboardConfigLayoutColumn</stache-code> object contains an array of <stache-code>SkyTileDashboardConfigLayoutTile</stache-code> objects that indicates the order of tiles. It has the following properties:
+      A <stache-code>SkyTileDashboardConfigLayout</stache-code> object that describes the tile dashboard's layout. The <stache-code>SkyTileDashboardConfigLayout</stache-code> object contains a <stache-code>singleColumn</stache-code> property of type <stache-code>SkyTileDashboardConfigLayoutColumn</stache-code> and a <stache-code>multiColumn</stache-code> property that is an array of type <stache-code> SkyTileDashboardConfigLayoutColumn</stache-code>. The <stache-code>singleColumn</stache-code> layout displays on small screens, while the <stache-code>multiColumn</stache-code> layout displays on larger screens. The <stache-code>SkyTileDashboardConfigLayoutColumn</stache-code> object contains an array of <stache-code>SkyTileDashboardConfigLayoutTile</stache-code> objects that indicates the order of tiles. It includes the following properties:
```

Comment:
There is an extra space in this portion of the text talking about the `multiColumn` property.

```
 is an array of type <stache-code> SkyTileDashboardConfigLayoutColumn</stache-code>
```

Reply:
@blackbaud-johnly the extra space is still here.

Reply:
Weird. I know I deleted it. Not sure how I messed that up, but I'll fix it now.

Reply:
Actually, I just opened the file again to remove the space, and I don't see the extra space anymore.